src/ontology/mondo-edit.obo

```diff
@@ -464918,6 +464920,2857 @@ is_a: MONDO:0005550 {source="https://orcid.org/0000-0002-5002-8648"} ! infectiou
+intersection_of: MONDO:0005583 {source="OMIA:001081"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0010679 {source="OMIA:001081"} ! cross-species analog Duchenne muscular dystrophy
+
+[Term]
+id: MONDO:1010200
+name: congenital muscular dystonia 1, non-human animal
+def: "Congenital muscular dystonia 1 that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001450 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001450"} ! non-human animal disease
+is_a: MONDO:1010196 {source="OMIA:001450", source="https://orcid.org/0000-0002-5002-8648"} ! congenital myopathy, non-human animal
+
+[Term]
+id: MONDO:1010201
+name: congenital pseudomyotonia, non-human animal
+def: "Congenital pseudomyotonia that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001464 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001464"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:001464"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0010977 {source="OMIA:001464"} ! cross-species analog Brody myopathy
+
+[Term]
+id: MONDO:1010202
+name: myotubular myopathy 1, non-human animal
+def: "Myotubular myopathy 1 that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001508 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001508"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:001508"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0010683 {source="OMIA:001508"} ! cross-species analog X-linked centronuclear myopathy
+
+[Term]
+id: MONDO:1010203
+name: polymyositis, non-human animal
+def: "Polymyositis that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001874 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001874"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0019127 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog polymyositis
+relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
+
+[Term]
+id: MONDO:1010205
+name: congenital increased muscular tonus, non-human animal
+def: "Congenital increased muscular tonus that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001978 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001978"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:001978"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0044682 {source="OMIA:001978"} ! cross-species analog MYBPC1-related autosomal recessive non-lethal arthrogryposis multiplex congenita syndrome
+
+[Term]
+id: MONDO:1010206
+name: muscular dystrophy, limb-girdle, type R6 (LGMDR6), non-human animal
```

Comment:
In human, "disease ~ type x" is based on the gene affected. Should we have only the species-specific term(s) instead of this generic term based on "general species unspecific" gene? 
Note that these species-specific diseases (based on species-specific genes) could be analogous to the human type x disease, but it is difficult to create an axiom referring to the general "species-not- specific" gene.